SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +156,20 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// Alternative connection method, when identifier and services are known for a peripheral.
+    ///
+    /// - Parameter peripheralUUID: The unique identifier (UUID) for the given peripheral.
+    /// - Parameter serviceUUIDs: The service UUIDs to for the given peripheral.
+    /// - Parameter timeout: The time in seconds before the connection attempt is stopped and the completion closure is called with a failed result.
+    /// - Parameter completion: The closures, called upon completion (succesful or otherwise) of the connection.
+    public func connect(peripheralUUID: UUID,
```

Comment:
The way that the connect method is currently implemented is definitely adding functionality that the CoreBluetooth APIs do not provide, as it is using the `retrievePeripherals()` method to check if it is already connected, which makes it difficult to follow Apple's Bluetooth best practices guidelines, unless one is okay with making multiple calls to underlying API methods. I can see why you took the liberty to add the connect method to the Peripheral, for situations where there is no direct access to the underlying CBCentralManager and CBPeripheral (which is all situations using this original repo) it makes things much cleaner.